guides/hosting/infrastructure/message-queue.md

```diff
@@ -41,6 +41,12 @@ You can configure the command just to run a certain amount of time and to stop i
 bin/console messenger:consume async --time-limit=60 --memory-limit=128M
 ```
 
+You can also configure the command to consume messages from multiple transports as it is recommended:
```

Comment:
also link to the symfony docs on multiple queues?

Reply:
I will add the same documentation regarding the prioritized transports here :+1: